stable/coredns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: coredns
-version: 1.1.1
-appVersion: 1.2.0
+version: 1.1.2
```

Comment:
This is a feature change, not a bugfix, so please bump minor version instead of patch.